content/docs/communities/moderation/_index.md

```diff
@@ -0,0 +1,114 @@
++++
+title = "Community Moderation"
+weight = 100
+[extra]
+emoji = "🛡️"
+tile = "I want to deploy moderation tools"
+last_updated = "2022-10-27T08:00:00Z"
++++
+
+## Power levels
+
+Matrix has a basic system of roles, sometimes also called Power Levels. It is a
+number usually going from 0 to 100. Advanced users can decide to use a different
+range, but we will not cover it here for the sake of simplicity.
+
+By default, there are three roles in Matrix:
+
+| Power Level | Role          |
+|-------------|---------------|
+| 0           | User          |
+| 50          | Moderator     |
+| 100         | Administrator |
+
+By default:
+
+- **Users** can send messages, media, reactions and redact their own messages in
+  a room.
+- **Moderators** can also change the room name, address, topic, remove users
+  from the room (temporarily or permanently), redact other's messages, send a
+  noisy notification to everyone in the room
+- **Administrators** can also change the history visibility (whether people can
+  see messages from before they joined or not), enable encryption in the room,
+  remove entire servers from the room, and promote others as Moderators or
+  Administrators
+
+Most people in your community are just going to be regular users. Usually
+Moderators are appointed to handle the ad-hoc moderation issues. Note that this
+can be an issue if you want to preserve the anonymity of your Trust & Safety
+team for their own security. The moderation bot mjolnir brings a good answer to
+this.
+
+If you're taking over the community of someone who was very technically versed
+into Matrix and strongly opinionated on power levels, the roles may not be the
+same. In that case, we recommend you to email
+[support@matrix.org](mailto:support@matrix.org) for help.
+
+## Your community's security guard
+
+The tools built-in Matrix are good for small group moderation, but they are
+limited for now: removing a user (temporarily or permanently) is only possible
+at the room level, and banning an entire server from a room is not exposed in
+the UI.
+
+Fortunately, a tool exists to step up your moderation game: a security guard
+robot you add to all of your public rooms, who can enforce your moderation
+decisions on all the rooms at once.
+
+And that's really what mjolnir is: a security guard for your rooms. Because of
+the very decentralised nature of Matrix, you need to add the robot to guard the
+door of each of your rooms, and you need to grant it the Administrator role so
```

Comment:
This is kinda bad, really (because you can't demote it again)..

Reply:
That's a very good point. In practice it depends:
- If mjolnir runs with an account you created, you can use the credentials to demote it
- If mjolnir runs with an account from a homeserver you have admin access to, you can use the admin API to demote it
- If you run a mjolnir instance with an account you don't have the credentials of, from a homeserver you don't have (indirect) admin access to, then indeed you can't demote it. This is going to be an issue when mjolnir for all is released for example.

The alternative is to update every room to lower several of the permissions from 100 to e.g. 99. It's quite cumbersome and error-prone. Worth discussing internally

Reply:
IIRC for Ruma I lowered a bunch of things to 50, but maybe it's not a good idea in general 😅